Doc/library/sys.rst

```diff
@@ -1060,6 +1062,31 @@ always available.
         This is still called as a fallback if a :data:`meta_path` entry doesn't
         have a :meth:`~importlib.abc.MetaPathFinder.find_spec` method.
 
+.. data:: module_names
+
+   A tuple of strings giving the names of standard library modules.
```

Comment:
```suggestion
   A tuple of strings containing the names of standard library modules.
```

Reply:
The text is copied from sys.builtin_module_names. I'm fine with changing it, but I would also suggest to change sys.builtin_module_names documentation in that case.

Reply:
As you prefer